lib/langchain/message.rb

```diff
@@ -34,6 +34,20 @@ def to_json(options = {})
   end
 
   class AIMessage < Message
+    def self.from_llm_response(llm_klass, llm_response)
```

Comment:
is it fine that Message know how to deal with each LLM?

Reply:
I guess another approach is to have each LLM specify a `completion_response_path` constant, something like the following:
```ruby
class Langchain::LLM::OpenAI
   COMPLETION_RESPONSE_PATH = ["choices", 0, "message"]

class Langchain::LLM::GooglePalm
   COMPLETION_RESPONSE_PATH = ["candidates", 0, "content"]
```
and then here you could just do:
```ruby
new llm_response.dig(**llm_klass::COMPLETION_RESPONSE_PATH)
```